### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NotebookQueryTable_ColorContrast: Color contrast ratio of "sys.tables"text  for Red  theme is below the expected ratio i.e 4.5:1

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5977]
[“Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
Tool Name: ColorContrastAnalyser
MAS Reference: MAS1.4.3

**Steps to Reproduce:**
1.Install the app Azure Data Studio.
2.Connect to the Database->Go to file->New Notebook.
3.Click on “Code” button and write query in the box for example SELECT * FROM sys.tables;
4.Open color contrast analyser and check the color contrast ratio.
5.It shows color contrast ratio less than expected color contrast ratio.

**Actual:**
Color contrast ratio for "sys.tables" text is 1.946:1.
And similar issue also observed with "select * from " text is 4.299:1 and for "1" text is 3.101:1 .

**Expected:**
The expected color contrast ratio  for text should be 4.5:1 ratio.

**User Impact:**
Low vision User will find it difficult to understand and will be confused if they will not able to see the buttons on this page.

**Recommendations:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:
MAS1.4.3-** https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D
**Attachment For Reference:**
![5977_ Accessibility A11y_AzureDataStudio_NotebookQueryTable_ColorContrast](https://user-images.githubusercontent.com/39204909/59268219-dbe23300-8c69-11e9-9be0-76f3f37d9f71.png)

![5977_ Accessibility A11y_AzureDataStudio_NotebookQueryTable_ColorContrast1](https://user-images.githubusercontent.com/39204909/59268586-a68a1500-8c6a-11e9-8aa8-49bd57786683.png)

![5977_ Accessibility A11y_AzureDataStudio_NotebookQueryTable_ColorContrast2](https://user-images.githubusercontent.com/39204909/59268857-42b41c00-8c6b-11e9-82cf-d24b82ba8eb6.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
For low vision users, we have the high contrast theme in Azure Data Studio, so we would suggest using that.

**Comment by chlafreniere** [comment]
Closing after conversation with accessibility testing team (no contrast requirements on non-default themes).
